src/error.rs

```diff
@@ -63,6 +65,21 @@ pub enum APIError {
     Timeout { domain: String },
 }
 
+#[derive(Debug, ThisError)]
+pub enum DevConfigError {
+    #[error("No config files found!")]
+    ConfigNotFound,
+    #[error("No dev secret config files found!")]
+    ConfigSecretNotFound,
+    #[error(
```

Comment:
Why this one is an error ? For me it's not an error, rather just an informative text.